### DISCUSSION TITLE
sms integration

### DISCUSSION THREAD

**Discussion by Jijiri89** [bryceandy/laravel-beem#2] [Q&A]
Please, Can I integrate this into laravel 10 project?

**Comment by bryceandy** [comment]
> Please, Can I integrate this into laravel 10 project?

Yes this works with Laravel 10

**Comment by Jijiri89** [comment]
> > Please, Can I integrate this into laravel 10 project?
> 
> Yes this works with Laravel 10

Thanks so much for your response.
Please what I need is the way to create a ui in the view (blade.php) where I can have a form to collect user inputs like name of the user to send sms to, message and the date of sechudeling the sms. How do I link that to the view? The sechudeled sms should be able to stored in the database. With this I know I have to get a model that can have, name, message body and schedule date of the message.
How I will be able to link the beem to laravel view is my problem now so kindly help me please. I really like your repository for beem sdk.
